### ISSUE TITLE
[Accessibility]A11y_VSCode_InteractiveEditorPlayground_ScreenReader:NVDA does not read the numbering of the written code.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52460]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details**

ScreenReader,NVDA[2018.2],JAWS[18.0.5038],Voiceover[MACOS High Sierra 10.13.5 Beta (17F70a)]
Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Navigate to Help(Alt+H) menu option and select "Interactive Editor Playground" from dropdown.
3. Navigate to editor  where code is written.
4. Check what NVDA is reading. 

Actual:
NVDA does not read the numbering of the written code.
Same issue observed in JAWS and Voiceover also.
Expected:
NVDA should read the numbering of the code.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
The user will not be able to get the numbering reference.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**
![accessibility a11y_vscode_interactiveeditorplayground_screenreader_nvda](https://user-images.githubusercontent.com/39204909/41655372-49197600-74aa-11e8-8893-90ff41a93537.png)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52429
